ons_alpha/topics/models.py

```diff
@@ -77,7 +77,7 @@ class TopicPage(BaseTopicPage):
     ]
     page_description = "A specific topic page. e.g. Public sector finance or Inflation and price indices"
 
-    headline_figures = StreamField([("figures", HeadlineFiguresBlock())], blank=True)
+    headline_figures = StreamField([("figures", HeadlineFiguresBlock())], blank=True, max_num=1)
```

Comment:
I hadn't realised the headline figures were in the topic page too - do we definitely need them there? They aren't in the design.